python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -2071,6 +2107,71 @@ async def wait_m1m3_settle(self) -> None:
         self.log.debug("Waiting for m1m3 to settle.")
         await asyncio.sleep(self.m1m3_settle_time)
 
+    def _map_m1m3_slew_setting_to_name(self, slew_setting: enum.IntEnum) -> str:
+        """
+        Map the slew setting enumeration to the corresponding key used
+        in the ``get_m1m3_slew_controller_settings`` method.
+
+        Parameters
+        ----------
+        slew_setting : enum.IntEnum
+            The slew setting enumeration value.
+
+        Returns
+        -------
+        str
+            The corresponding key used in the get method.
+        """
+        mapping = {
+            MTM1M3.SetSlewControllerSettings.ACCELERATIONFORCES: "ACCELERATIONFORCES",
+            MTM1M3.SetSlewControllerSettings.BALANCEFORCES: "BALANCEFORCES",
+            MTM1M3.SetSlewControllerSettings.BOOSTERVALVES: "BOOSTERVALVES",
+            MTM1M3.SetSlewControllerSettings.VELOCITYFORCES: "VELOCITYFORCES",
+        }
+        if slew_setting not in mapping:
+            raise ValueError(f"Invalid slew setting: {slew_setting}")
+        return mapping[slew_setting]
+
+    async def set_m1m3_slew_controller_settings(
+        self, slew_setting: enum.IntEnum, enable_slew_management: bool
+    ) -> None:
+        """
+        Set a specific M1M3 slew controller setting based on the provided
+        enumeration.
+
+        Parameters
+        ----------
+        slew_setting : enum.IntEnum
+            The specific force component setting to be changed.
+        enable_slew_management : bool
+            True to enable, False to disable the specified force component
+            controlled by the slew controller.
+        """
+        setting_key = self._map_m1m3_slew_setting_to_name(slew_setting)
```

Comment:
I don't think you need this method. You can get the "name" of the enumeration by using:

```
slew_setting.name
```

Reply:
I removed the map method here, but I needed to create a different version of it to deal with mock methods.